h2o-py/tests/testdir_algos/coxph/pyunit_pubdev_8945_coxph_heart_interactions_catNum_catCat_NumNum_NA_mojo.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import tempfile
+
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.coxph import H2OCoxProportionalHazardsEstimator
+
+# this test makes sure the enum x enum, enum x num, num x num interactions work for coxph mojo
+def test_coxph_enum_num_interactions_orders(interaction_orders):
+    # bunch of random columns to be added to the dataset
+    data = h2o.import_file(pyunit_utils.locate("smalldata/coxph_test/heart_random_num_enum_NAs.csv"))
+    data['surgery'] = data['surgery'].asfactor()
+    data['transplant'] = data['transplant'].asfactor()
+
+    model = H2OCoxProportionalHazardsEstimator(start_column="start", stop_column="stop", 
+                                               interaction_pairs=interaction_orders)
+    model.train(x=["age", "surgery", "transplant", "C1", "C2", "C3", "C4"], y="event", 
+                training_frame=data)
+    pred_h2o = model.predict(data)
+    # export new file (including the random columns)
+    sandbox_dir = tempfile.mkdtemp()
+    model.download_mojo(path=sandbox_dir)
+    mojo_name = pyunit_utils.getMojoName(model._id)
+    input_csv = "%s/in.csv" % sandbox_dir
+    h2o.export_file(data, input_csv)
+    x, pred_mojo = pyunit_utils.mojo_predict(model, sandbox_dir, mojo_name)
+    pyunit_utils.compare_frames_local(pred_h2o, pred_mojo, 1, tol=1e-10)
+    
+def test_coxph_enum_num_interactions():
```

Comment:
- [pyunit_pubdev_8945_coxph_heart_interactions_catNum_catCat_NumNum_NA_mojo.py](https://github.com/h2oai/h2o-3/pull/6677/files#diff-b3175df5e53734ab1195124d6864c00ea1ba105b41b43cf917ab8a47283f74df)
- [pyunit_pubdev_8945_coxph_heart_interactions_catNum_catCat_NumNum_mojo.py](https://github.com/h2oai/h2o-3/pull/6677/files#diff-928497cad043220e177ace11316fea4bdd5c0cc13211f7392e87d1972d41db4d)

Seems to be identical except the training dataset. Can we merge it one file and make the `test_coxph_enum_num_interactions` method parametrized with the traning file?

Reply:
It seems the same merge of tests could be done between:
- [pyunit_pubdev_8945_coxph_heart_interactions_cat_cat_NA_mojo.py](https://github.com/h2oai/h2o-3/pull/6677/files#diff-5c98549d1dff14535cecef6e94a41f55979e2f215c900ac7171204e43820e2d4)
- [pyunit_pubdev_8945_coxph_heart_interactions_cat_cat_mojo.py](https://github.com/h2oai/h2o-3/pull/6677/files#diff-c1a27dd19583c6194e9ad760aa3f2c2a596aafe2d15f017308c0e8e8a1ccbe29)

and also between:
- [pyunit_pubdev_8945_coxph_heart_interactions_cat_num_mojo.py](https://github.com/h2oai/h2o-3/pull/6677/files#diff-75ca079d481cf524bc0c3879e15da16f532813728034953d7d462598c325bd35)
- [pyunit_pubdev_8945_coxph_heart_interactions_cat_num_NA_mojo.py](https://github.com/h2oai/h2o-3/pull/6677/files#diff-1e536385ef28680b25be3fbec8d1f408ba23317516a86bc8abeb1362201afc9c)

Reply:
Done.  Tests combined.